verify_ehc.py

```diff
@@ -794,18 +804,35 @@ def download_fr_certs(token: Optional[str] = None) -> CertList:
     return certs
 
 def download_nl_certs(token: Optional[str] = None) -> CertList:
+    # Fetch the root certificate for the Netherlands; used to secure the
+    # trust list. Non fatal error if this fails.
+    response = requests.get(ROOT_URL_NL, headers={'User-Agent': USER_AGENT})
+    cacert = None
+    try:
+        if response.status_code == 200:
+          cacert = load_der_x509_certificate(response.content)
+    except:
+        pass
```

Comment:
`except: pass` is not good, error should be handled/logged, see `download_de_certs()`.